specs/IR-0003/suite/test_constitution.py

```
@@ -0,0 +1,124 @@
+        for plane in ("声明面", "执行面", "判定面"):
+            self.assertIn(plane, body, f"14c 缺面: {plane}")
+        self.assertIn("判定锚点永不外置", body)
+        self.assertIn("outbound-only", body)
+        self.assertIn("label 唯一真源", body)
+
+    def test_section5_hard_predicate_untouched(self):
+        """§5 硬谓词+shadow 核心句逐字在位（ADR-0103 决策 1 硬边界）。"""
+        t = constitution_text()
+        s5 = section(t, "5", "6")
+        for kw in (
+            "硬谓词白名单（fail-closed）+ 常设 shadow 模式",
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)